distributed/scheduler.py

```diff
@@ -116,8 +153,12 @@ def cclass(cls):
 
 
 LOG_PDB = dask.config.get("distributed.admin.pdb-on-err")
-DEFAULT_DATA_SIZE = parse_bytes(
-    dask.config.get("distributed.scheduler.default-data-size")
+DEFAULT_DATA_SIZE = declare(
+    Py_ssize_t, parse_bytes(dask.config.get("distributed.scheduler.default-data-size"))
+)
+UNKNOWN_TASK_DURATION = declare(
+    double,
+    parse_timedelta(dask.config.get("distributed.scheduler.unknown-task-duration")),
 )
```

Comment:
Ideally we would annotate these and that would be sufficient. Unfortunately that didn't appear to work ( https://github.com/cython/cython/issues/3944 ). So used `declare` here instead, which does handle this correctly.